README.md

```diff
@@ -65,14 +65,12 @@ The [CircleCI configuration file](dist/circleci/.circleci/config.yml) uses a
 [custom Docker image](https://hub.docker.com/r/juampynr/drupal8ci/) that extends from
 the [official Drupal image](https://hub.docker.com/_/drupal/) and it is [hosted at
 Docker Hub](https://hub.docker.com/r/juampynr/drupal8ci/). If this image
-does not fit your project's architecture, then have a look at its
-[Dockerfile](https://github.com/Lullabot/drupal8ci/blob/master/circleci/.circleci/images/primary/Dockerfile)
-and consider [creating your own image](https://circleci.com/docs/2.0/custom-images/)
+does not fit your project's architecture then consider [creating your own image](https://circleci.com/docs/2.0/custom-images/)
 based out of it.
    
 #### Setting up the update path
 
-The Behat job requires a running Drupal 8 site. The repository contains the code, but for running
+The Behat job requires a running Drupal 9 site. The repository contains the code, but for running
```

Comment:
@fjgarlin I know that you like Crypress.io (https://events.drupal.org/europe2020/sessions/add-end-end-tests-your-drupal-site-one-hour-cypressio). I love it. I would love to replace the Behat tests by Cypress.io ones. Will create an issue for it.

Reply:
Here it is https://github.com/Lullabot/drupal9ci/issues/67

Reply:
Great!